installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,166 @@
+### Configure the newly installed drivers in odbcinst.ini ###
+#############################################################
+# Create the ODBC directory if it doesn't exist already
+if [[ ! -e $ODBC_PATH ]]; then
+    info "Creating $ODBC_PATH directory"
+    mkdir -p $ODBC_PATH
+elif [[ ! -d $ODBC_PATH ]]; then
+     err "$ODBC_PATH already exists but is not a directory"
+     exit 1   
+fi
+
+#Update the odbcinst.ini with the latest driver path and mark it as installed
+if [[ ! -e $INI ]]; then
+    info "$INI does not exists, creating"
+    touch $INI
+    echo "[ODBC Drivers]" >> $INI
+    echo "$MDB = Installed" >> $INI
+    echo "" >> $INI
+    echo "[$MDB]" >> $INI
+    echo "Driver = $MDB_DRIVERS" >> $INI
+    echo "" >> $INI
+    $CREATED_NEW_INI_FILE=1
+fi
+
+#If there was an existing odbcinst file, update the odbcinst.ini
+# with the latest driver path
+if [[ $CREATED_NEW_INI_FILE == 0 ]]; then
+    BACK_INI="$INI.back"
+    info "Back up file $BACK_INI"
+    ODBC_DRIVERS_SECTION_EXISTS=0
+    CURRENT_SECTION="NONE"
+    PREVIOUS_SECTION="NONE"
+
+    # Make a copy of the existing ini file
+    cp $INI $BACK_INI
+    if [[ -e $BACK_INI ]]; then
+        # Clear the INI file
+        > $INI
+
+        # Read the backup odbcinst.ini.back file and recreate the new updated odbcinst.ini one
+        while IFS= read -r line
+        do
+            if [[ "$line" =~ ^\[.*\]$ ]]; then
+                PREVIOUS_SECTION=$CURRENT_SECTION
+                # Found a section
+                if grep -iqF "[ODBC Drivers]" <<< "$line"; then
+                    # Found the [ODBC Drivers] section 
+                    # where all drivers are listed
+                    ODBC_DRIVERS_SECTION_EXISTS=1
+                    CURRENT_SECTION="ODBC"
+                elif grep -iqF "[$MDB]" <<< "$line"; then
+                    # Found the [AtlasSQL Unicode ODBC {version}] section
+                    CURRENT_SECTION="UNICODE"
+                else
+                    CURRENT_SECTION="OTHER"
+                fi
+
+                # Found a new section right after [ODBC Drivers]
+                # Need to add the drivers lines in [ODBC Drivers] section
+                if [ $PREVIOUS_SECTION == "ODBC" ]; then
+                    flagDriversAsInstalled
+                    echo "" >> $INI
+                fi
+                echo "$line" >> $INI
+            else
+                if [ $CURRENT_SECTION == "ODBC" ]  && grep -iqF "$MDB" <<< "$line" ; then
+                    # Found the latest unicode MongoDB driver already configured
+                    # Make sure to mark it "Installed"
+                    echo "$MDB = Installed" >> $INI
+                    MDB_DRIVER_INSTALLED=1
+                elif [ $CURRENT_SECTION == "UNICODE" ] && grep -iqF "Driver" <<< "$line" ; then
+                    # Make sure the driver install path is correct
+                    echo "Driver = $MDB_DRIVERS" >> $INI
+                    MDB_DRIVER_CONFIGURED=1
```

Comment:
We also need this key-value pair in the ini for Mac : `DriverUnicodeType = utf16`

Reply:
Would that first one not force all drivers to use utf16 on the system?

Reply:
Yes it would.  Let's leave the `AppUnicodeType` out then and keep the `DriverUnicodeType = utf16`.